doc/modules/neural_networks_supervised.rst

```diff
@@ -0,0 +1,357 @@
+.. _neural_network:
+
+==================================
+Neural network models (supervised)
+==================================
+
+.. currentmodule:: sklearn.neural_network
+
+
+.. _multilayer_perceptron:
+ 
+Multi-layer Perceptron
+======================
+
+**Multi-layer Perceptron (MLP)** is a supervised learning algorithm that learns
+a function :math:`f(\cdot): R \rightarrow R` by training on a dataset. Given a 
+set of features :math:`X = {x_1, x_2, ..., x_m}` and a target :math:`y`, it 
+can learn a non-linear, complex functions for either classification
+or regression. It is different from logistic regression, in that between the input 
+and the output layer, there can be one or more hidden layers. Figure 1 shows 
```

Comment:
one or more hidden layers -> one of more non-linear layers, called hidden layers.